src/main/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchSqlGenerationVisitor.java

```diff
@@ -0,0 +1,53 @@
+package com.exasol.adapter.dialects.elasticsearch;
+
+import com.exasol.adapter.AdapterException;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.sql.SqlColumn;
+
+/**
+ * Implementation of {@link SqlGenerationVisitor} for generating ElasticSearchSQL queries.
+ */
+public class ElasticSearchSqlGenerationVisitor extends SqlGenerationVisitor {
+
+    /**
+     * Construct a new instance of {@link ElasticSearchSqlGenerationVisitor}.
+     *
+     * @param dialect SQL dialect
+     * @param context SQL generation context
+     */
+    public ElasticSearchSqlGenerationVisitor(final SqlDialect dialect, final SqlGenerationContext context) {
+        super(dialect, context);
+    }
+
+    @Override
+    public String visit(final SqlColumn column) throws AdapterException {
+        return this.getTablePrefix(column) + this.getColumnName(column);
```

Comment:
The prefix in this case contains the separator implicitly. Make it explicit, please.